File: Product_APP_MVC/app/src/main/java/com/example/product_app_mvc/MainActivity2.java
package com.example.product_app_mvc;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.example.product_app_mvc.AllProducts.Viewer.Viewer.MealDetailsActivity;
import com.example.product_app_mvc.FavProducts.View.FavProductActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity2 extends AppCompatActivity {

    Animation animation;
    TextView txtView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main2);

        animation = AnimationUtils.loadAnimation(this,R.anim.animation_layout);
        txtView =findViewById(R.id.textView2);
        txtView.startAnimation(animation);
        txtView.postOnAnimationDelayed(new Runnable() {
            @Override
            public void run() {
                Intent outIntent = new Intent(MainActivity2.this, HomeActivity.class);
                startActivity(outIntent);
                finish();
            }},3000);

    }
}

File: Android_Development/05_Android_Kotlin/05_Session/viewbinding/fragments/app/src/main/java/com/example/fragments/MainActivity2.kt
package com.example.fragments

import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import com.example.fragments.databinding.ActivityMain2Binding // Import your generated binding class

class MainActivity2 : AppCompatActivity() {

    // Declare the binding variable
    private lateinit var binding: ActivityMain2Binding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()

        // Initialize the binding variable
        binding = ActivityMain2Binding.inflate(layoutInflater)
        setContentView(binding.root)


    }
}
